builtin/providers/aws/resource_aws_elb.go

```diff
@@ -438,6 +467,22 @@ func resourceAwsElbUpdate(d *schema.ResourceData, meta interface{}) error {
 			},
 		}
 
+		logs := d.Get("access_logs").(*schema.Set).List()
+		if len(logs) > 0 {
```

Comment:
I believe the check here is just to see if we've found `access_logs` in the configuration. `access_logs` is a set and _technically_ it's valid to have more than one in the config (from a HCL standpoint), but on AWS side yes it's likely an error. You can see later in the code only `logs[0]` is used.